main.py

```diff
@@ -60,11 +60,14 @@ def create_ui_components(root):
     update_interval_slider.set(2)
     update_interval_slider.grid(row=3, column=1, padx=10, pady=10, sticky="nsew")
 
-    return transcript_textbox, response_textbox, update_interval_slider, update_interval_slider_label, freeze_button
+    combobox = ctk.CTkOptionMenu(root, values=list(LANGUAGES.values()))                                  
```

Comment:
Could we move the dropdown somewhere  better in the UI. We are added alot of white space here.